go/kbfs/libkbfs/keybase_service_base.go

```diff
@@ -1171,56 +1166,20 @@ func (k *KeybaseServiceBase) NotifyOnlineStatusChanged(ctx context.Context,
 
 // Notify implements the KeybaseService interface for KeybaseServiceBase.
 func (k *KeybaseServiceBase) Notify(ctx context.Context, notification *keybase1.FSNotification) error {
-	// Reduce log spam by not repeating log lines for
```

Comment:
why delete all this stuff?

Reply:
This is the fourth bullet point in this PR's description:

> We no longer log file paths when sending notifications up to the service; these are just plaintext strings so they'd be hard to obfuscate, and it doesn't seem worthwhile.

It didn't seem worth all the plumbing needed to get an obfuscator here, for something that's very rarely useful and usually just clutter.  The filename here is a full path, like `/keybase/private/strib/a/b/c/`, so obfuscating it would require breaking it up and obfuscating just the last pieces, and it just didn't seem worth it.